.github/ISSUE_TEMPLATE/new_issue.yaml

```
@@ -1,5 +1,5 @@
 name: New Issue
-description: Report something that doesn't fit into above categories
+description: Report something that doesn't fit into another category
```

This doesn't sound right to me "another category" what category? I think "doesn't fit in any other category" sounds better

```suggestion
description: Report something that doesn't fit in any other category
```